### DISCUSSION TITLE
Where are stocked PDF print in a Qfield cloud project ?

### DISCUSSION THREAD

**Discussion by GlaDal** [opengisch/QField#3864] [Q&A]
All is on the title ! :-)
This PDF print are accessible on Qfield project data file.  Is there a way to access on the PDF files from another way (not in Qfield but on the tablet files or maybe on the website Qfield cloud on our account ?)
Thanks

**Comment by meyerlor** [comment]
I had the same Question and opened a ticket via Qfield Cloud. The answer was that at the moment it is not possible that those PDFs are synced into the cloud.
The only way to access them is to go in Qfield into your Settings-> Open Project folder -> Select PDF via long-tap (or the folder where you export your PDFs to) -> Export to...

I export the folder into my Documents folder, from there the PDFs are openly accessable
